<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PID_Widget</class>
 <widget class="QWidget" name="PID_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>480</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки ПИД</string>
  </property>
  <widget class="QLabel" name="label_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал по высоте:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал по высоте:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал поворота через крен:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал поврота через крен:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал для тонгажа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал для крена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал для рысканья:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал для тонгожа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал для крена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал для рысканья:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Get_PID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить ПИД</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Set_PID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отгрузить ПИД</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть </string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_P_Alt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2_D_Alt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_3_P_Bank">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_4_D_Bank">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_5_P_Pitch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_6_P_Roll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_7_P_Yaw">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_8_D_Pitch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_9_D_Roll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_10_D_Yaw">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del_pid">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить ПИД</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
